### PR TITLE
Docs: rename Enterprise to self-hosted

### PR DESCRIPTION
**Pull request by wissem**

### CODE REVIEW

File: docs/self-hosted/installation.md [hunk 1]
```diff
@@ -19,67 +19,67 @@
...
+![Upload License](/Resources/public/img/docs/self-hosted/03-upload-license.png)
 
 Once validated, you will be asked to set a password for the management console or to protect it with a different authentication mechanism.
 
-![Security Console](/Resources/public/img/docs/enterprise/04-secure-console.png)
+![Security Console](/Resources/public/img/docs/self-hosted/04-secure-console.png)
 
 ### Server Settings
-In the management console you will have to configure a few options before you can start the Private Packagist Enterprise application.
+In the management console you will have to configure a few options before you can start the Private Packagist Self Hosted application.
```
Comment (glaubinix) [inline docs/self-hosted/installation.md:53]:
```suggestion
In the management console you will have to configure a few options before you can start the Private Packagist Self-Hosted application.
```

File: docs/self-hosted/installation.md [hunk 2]
```diff
@@ -19,67 +19,67 @@
...
 
 #### E-mail
 Select options for your SMTP server or use the GMail option for a simplified form if want to send email through GMail. This step is required to allow Private Packagist to send transactional email to users.
 
-![Email Settings](/Resources/public/img/docs/enterprise/05-email-settings.png)
+![Email Settings](/Resources/public/img/docs/self-hosted/05-email-settings.png)
 
 ## Application Startup
-Save and apply the settings to start Private Packagist Enterprise. Leave the Setup mode enabled.
+Save and apply the settings to start Private Packagist Self Hosted. Leave the Setup mode enabled.
```
Comment (glaubinix) [inline docs/self-hosted/installation.md:61]:
```suggestion
Save and apply the settings to start Private Packagist Self-Hosted. Leave the Setup mode enabled.
```

File: docs/self-hosted/installation.md [hunk 3]
```diff
@@ -19,67 +19,67 @@
...
 
-![Start Application](/Resources/public/img/docs/enterprise/06-save-start.png)
+![Start Application](/Resources/public/img/docs/self-hosted/06-save-start.png)
 
-Once the application has started, click on the open link to access the Private Packagist Enterprise Web Interface.
+Once the application has started, click on the open link to access the Private Packagist Self Hosted Web Interface.
```
Comment (glaubinix) [inline docs/self-hosted/installation.md:65]:
```suggestion
Once the application has started, click on the open link to access the Private Packagist Self-Hosted Web Interface.
```

File: docs/self-hosted/installation.md [hunk 4]
```diff
@@ -19,67 +19,67 @@
...
 
 In case the HTTP proxy should not be used to access certain domains or IPs then you can configure such
 a NO_PROXY list in the Private Packagist application in the Admin Panel under Global Configuration.
 
-![No HTTP Proxy](/Resources/public/img/docs/enterprise/07-02-http-proxy-no-proxy.png)
+![No HTTP Proxy](/Resources/public/img/docs/self-hosted/07-02-http-proxy-no-proxy.png)
 
 ## Authentication Setup
-Within Private Packagist Enterprise you now need to set up at least one user authentication method. You have the choice between authentication with email addresses and passwords and several OAuth integrations with third party services.
+Within Private Packagist Self Hosted you now need to set up at least one user authentication method. You have the choice between authentication with email addresses and passwords and several OAuth integrations with third party services.
```
Comment (glaubinix) [inline docs/self-hosted/installation.md:82]:
```suggestion
Within Private Packagist Self-Hosted you now need to set up at least one user authentication method. You have the choice between authentication with email addresses and passwords and several OAuth integrations with third party services.
```

File: features/integration-github-bitbucket-gitlab.md [hunk 5]
```diff
@@ -57,7 +57,7 @@ Private Packagist integrates with the following systems:
 * Webhooks: Code changes and releases
 
 #### GitLab (Self-hosted)
-* OAuth: Users authenticate on Private Packagist with their GitLab accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/gitlab-integration-setup.md) for Enterprise.
+* OAuth: Users authenticate on Private Packagist with their GitLab accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/self-hosted/gitlab-integration-setup.md) for Self.
```
Comment (glaubinix) [inline features/integration-github-bitbucket-gitlab.md:60]:
```suggestion
* OAuth: Users authenticate on Private Packagist with their GitLab accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/self-hosted/gitlab-integration-setup.md) for Self-Hosted.
```

**Review by glaubinix — CHANGES_REQUESTED** [review]
All the screenshots still have packagist-enterprise.com.lo as URL (not sure if we want to change) but the once showing Private Packagist also say Enterprise in the top navigation bar

File: features/integration-github-bitbucket-gitlab.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ Private Packagist integrates with the following systems:
 * Webhooks: Code changes, releases, created repositories, team creation or member changes
 
 #### GitHub Enterprise
-* OAuth: Users authenticate on Private Packagist with their GitHub accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/github-integration-setup.md) for enterprise.
+* OAuth: Users authenticate on Private Packagist with their GitHub accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/self-hosted/github-integration-setup.md) for self-hosted.
```
Comment (glaubinix) [inline features/integration-github-bitbucket-gitlab.md:30]:
```suggestion
* OAuth: Users authenticate on Private Packagist with their GitHub accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/self-hosted/github-integration-setup.md) for Self-Hosted.
```

File: features/integration-github-bitbucket-gitlab.md [hunk 7]
```diff
@@ -41,7 +41,7 @@ Private Packagist integrates with the following systems:
 * Webhooks: Code changes and releases
 
 #### Bitbucket Server / Stash
-* OAuth: Users authenticate on Private Packagist with their Bitbucket Server accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/bitbucket-server-integration-setup.md) for enterprise.
+* OAuth: Users authenticate on Private Packagist with their Bitbucket Server accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/self-hosted/bitbucket-server-integration-setup.md) for self-hosted.
```
Comment (glaubinix) [inline features/integration-github-bitbucket-gitlab.md:44]:
```suggestion
* OAuth: Users authenticate on Private Packagist with their Bitbucket Server accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/self-hosted/bitbucket-server-integration-setup.md) for Self-Hosted.
```

**Comment by glaubinix** [comment]
There are some screenshots that are still referring the old URL
Screenshot 2021-11-05 at 11 44 49
Screenshot 2021-11-05 at 11 45 43
Screenshot 2021-11-05 at 11 45 53

Did you do something to pictures 2-6 on the installation guide? If you look at the browser bar, somehow it feels like those pictures are very squished together? Like the nav bar is using a different resolution to the rest of the picture
Screenshot 2021-11-05 at 11 49 04

**Comment by wissem** [comment]
@glaubinix updated for all images including the squished ones

**Review by glaubinix — COMMENTED** [review]
Resources/public/img/docs/self-hosted/01-ssl-warning.png shows the wrong URL. Should be the one with the port 8800 see https://packagist.com/docs/enterprise/installation#replicated-setup

**Review by glaubinix — APPROVED** [review]

**Pull request merged**